core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataManagerTest.scala

```diff
@@ -2051,4 +2055,59 @@ class GroupMetadataManagerTest {
     group.transitionTo(CompletingRebalance)
     expectMetrics(groupMetadataManager, 1, 0, 1)
   }
+
+  def addDelay(durationMs: Long)(groupMetadata: GroupMetadata): Unit ={
```

Comment:
nit: since we only use this from `testPartitionLoadMetric`, does it makes sense to move it inside that test case? Also, missing space after `=`.

Reply:
Removed entirely since the tests have changed.